### ISSUE TITLE
Make features of plugin optional

### ISSUE THREAD

**Issue by Tyler799** [Toofifty/easy-giantsfoundry#6]
I find some features like the mould optimizer and actions worth of heat/actions to stage complete very useful - but a lot of the rest clutter the screen a bit and I don't use at all. (The percentages and object highlights)

Would it be possible to make some features optional with a toggle?

**Comment by Raisoshi** [comment]
I agree, "Heat" and "Stage" are much better to look at using the official UI. Object highlights I like though.
